CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- added in colour variables for discovery and live badges and banners
+- updated link colour variables to match gov.uk [#795](https://github.com/hmrc/assets-frontend/pull/795)
```

Comment:
@gavinwye I'd be inclined to agree with you if we were referring to user needs on the frontend of things, but as far as changelog updates commits are concerned, there's already style and consistency in place with regards to HMRC AF changelogs and I'm happy to comply with them.